ecs/marshall.go

```diff
@@ -39,7 +39,8 @@ func marshall(template *cloudformation.Template) ([]byte, error) {
 		if resources, ok := input["Resources"]; ok {
 			for _, uresource := range resources.(map[string]interface{}) {
 				if resource, ok := uresource.(map[string]interface{}); ok {
-					if resource["Type"] == "AWS::ECS::TaskDefinition" {
+					switch resource["Type"] {
+					case "AWS::ECS::TaskDefinition":
```

Comment:
changing this for a switch, because you foresee we'll have other cases ? (just curious)

Reply:
yes indeed. goformation can't handle negative boolean (default=true)
For testing purpose I added a few other cases. Using a switch then has less impacts